tests/test_coordinator.py

```
@@ -1505,5 +1520,70 @@ def fake_async_create_issue(**kwargs: Any) -> None:
+    [
+        pytest.param({}, id="missing"),
+        pytest.param({"device_unique_id": ""}, id="blank"),
+        pytest.param({"device_unique_id": None}, id="none"),
+        pytest.param({"device_unique_id": 123}, id="number"),
+        pytest.param({"device_unique_id": "   "}, id="whitespace"),
+    ],
+)
+async def test_check_device_unique_id_invalid_runtime_id_is_not_counted(
+    monkeypatch: pytest.MonkeyPatch,
+    make_config_entry: Callable[..., MockConfigEntry],
+    fake_client: Any,
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `fake_client`

(ANN401)